src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/Properties/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="context_menu_copy" xml:space="preserve">
     <value>Copy (Enter)</value>
   </data>
+  <data name="convert_from_ounce" xml:space="preserve">
+    <value>From {0}</value>
```

Comment:
```suggestion
    <value>From {0}</value>
    <comment>Like "Convert from unit".</comment>
```

Reply:
@htcfreek Doesn't the name of the field clears that up?

Reply:
But still resolved it.